Changing hpmn to dashevo in dash core (#24)
dashpay/docs-core: docs/api/remote-procedure-calls-evo.md

| → →<br>`votingAddress`         | string       | Required<br>(exactly 1) | The voting address                                                                                                                               |
| → →<br>`isValid`               | bool         | Required<br>(exactly 1) | Set to `true` if masternode is valid                                                                                                             |
| → →<br>`nVersion`              | number       | Required<br>(exactly 1) | **Added in Dash Core 19.0.0**<br>BLS version<br>`1` - Legacy BLS scheme<br>`2` - [Basic BLS scheme](https://github.com/dashpay/dash/issues/5001) |

> | → →<br>`nType`                 | number       | Required<br>(exactly 1) | **Added in Dash Core 19.0.0**<br>Type of masternode<br> `0` - Regular masternode<br>`1` - Evolution masternode                            |
> | → →<br>`platformHTTPPort`      | number       | Optional<br>(0 or 1)    | **Added in Dash Core 19.0.0**<br>TCP port of Platform HTTP/API interface (EvoNodes only)                                                            |
> | → →<br>`platformNodeID`        | string (hex) | Optional<br>(0 or 1)    | **Added in Dash Core 19.0.0**<br>Platform P2P node ID, derived from P2P public key (EvoNodes only)                                                  |

Reviewer: ```suggestion
| → →<br>`nType`                 | number       | Required<br>(exactly 1) | **Added in Dash Core 19.0.0**<br>Type of masternode<br> `0` - Regular masternode<br>`1` - Evolution masternode                            |
| → →<br>`platformHTTPPort`      | number       | Optional<br>(0 or 1)    | **Added in Dash Core 19.0.0**<br>TCP port of Platform HTTP/API interface (evonodes only)                                                            |
| → →<br>`platformNodeID`        | string (hex) | Optional<br>(0 or 1)    | **Added in Dash Core 19.0.0**<br>Platform P2P node ID, derived from P2P public key (evonodes only)                                                  |
```

}
```

> ### ProTx Update Service EvoNode

Reviewer: The heading here is basically the RPC name with no underscores so let's leave them HPMN
```suggestion
### ProTx Update Service HPMN
```

> The "protx update_service_hpmn" RPC creates and sends a ProUpServTx to the network. This will update the IP address and the Platform fields of a EvoNode. If this is done for a EvoNode that got PoSe-banned, the ProUpServTx will also revive this EvoNode.

Reviewer: ```suggestion
The "protx update_service_hpmn" RPC creates and sends a ProUpServTx to the network. This will update the IP address and the Platform fields of an evonode. If this is done for an evonode that was PoSe-banned, the ProUpServTx will also revive this evonode.
```
